ceos/src/components/friendList/FavFriends/FavProfile.tsx

```diff
@@ -0,0 +1,27 @@
+import { ProfileImg } from "@components/common/ProfileImg";
+import { columnFlex } from "style/commonStyle";
+import styled from "styled-components";
+import { friendListType } from "types/friendListType";
+
+type FavProfileProps = Pick<friendListType, "profile" | "user">;
+
+export default function FavProfile(props: FavProfileProps) {
+  const { profile, user } = props;
```

Comment:
함수 인자에서 바로 props를 구조 분해 할당하면 어떨까요!
```suggestion
export default function FavProfile({ profile, user }: FavProfileProps) {
```